### PR TITLE
Addition to a troubleshooting link

### PR DESCRIPTION
**Pull request by joinimran**
Added a link to a troubleshooting document as suggested by the user.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6620

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/raw-data-export-event-hub.md [hunk 1]
```diff
@@ -91,3 +91,4 @@ To get the data types for event properties do the following:
 - [Microsoft Defender for Endpoint streaming API](raw-data-export.md)
 - [Stream Microsoft Defender for Endpoint events to your Azure storage account](raw-data-export-storage.md)
 - [Azure Event Hubs documentation](https://docs.microsoft.com/azure/event-hubs/)
+- [Troubleshoot connectivity issues - Azure Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/troubleshooting-guide)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/raw-data-export-event-hub.md:94]:
```suggestion
- [Troubleshoot connectivity issues - Azure Event Hubs](https://docs.microsoft.com/azure/event-hubs/troubleshooting-guide)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the locale from the link.

**Comment by illfated** [comment]
Fair point. Language agnostic link https://docs.microsoft.com/azure/event-hubs/troubleshooting-guide tested, shows English language (with correct link locale) if my locale language is not available for the page. (This language agnostic test is very useful when suggesting new links or link changes in a PR.)

**Comment by joinimran** [comment]
Thanks @JohanFreelancer9 - I missed this.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**